### ISSUE TITLE
Download latest update on cellphone?

### ISSUE THREAD

**Issue by shinimaru** [suriyun-production/mmorpg-kit-docs#1487]
As the title says I wanted to know if there was a way I could download the most recent version of the asset pack to my cellphone? I won't have internet for my PC for a good while, so I can't download it the way I'm supposed to. Being broke sucks

I included the pictures as possible a way to prove I already bought it way back in 2020. And my last update was in November 2021. 

I just want the inventory loss system and any other new goodies so I can continue development.

Can anyone help me with this weird issue?

![IMG_20220301_132346085.jpg](https://user-images.githubusercontent.com/71353444/156227590-ea2ce0e9-73b3-4c12-8e7b-2d98f4781009.jpg)

![Screenshot_20220301-132713.png](https://user-images.githubusercontent.com/71353444/156227613-916eba6d-459a-4b25-beb4-654cae956117.png)

**Comment by insthync** [comment]
You may try turning on the hotspot from your phone then use your PC to connect to it.

**Comment by shinimaru** [comment]
Thank you. I'm never dealt with discord before but I guess it's time to start.

**Comment by shinimaru** [comment]
Well I went on discord. Not sure what I was looking at the chat looked like a video game chat heard towards teens with everyone trolling each other. Not sure I would get any help there. Why would they help me? Never understood the purpose of discord and I still don't.

**Comment by moepi2k** [comment]
what u talking about. there was never a question of a new user since yesteday. so whats ur name in discord? it was really quiet the last few days

**Comment by Randolphjand** [comment]
U can use your phone to give your pc internet with direct usb cable from phone to pc and there is a setting on the phone to allow your pc to use it.

**Comment by shinimaru** [comment]
About usb tethering. Att treasure hotspot and usb tethering as the same thing as far as an addition to the service is concerned. . I have to pay extra for them and I can't rn.

And as far as discord. I signed up and got into the group that was suggested in the earlier post, and honestly I didn't know what I was looking at. I clicked where it said chat in the group and it seemed like some guys where playing a video game and using that chat scroll to talk shit to each other. I'll check to make sure I didn't mispell it or maybe it auto-completed  to a different name when I typed it in the search and I didn't notice.

Thanks for the follow up btw, I'm "Shinimaru" on discord as well.
